### ISSUE TITLE
Attributes Temp2 and Temp3 not updated

### ISSUE THREAD

**Issue by hardion** [MaxIV-KitsControls/Libera-SinglePassE#23]
LiberaSinglePassE::read_Temp2() failed get value test/itech/libera-spe4
![screenshot_01](https://f.cloud.github.com/assets/305207/150010/90d0f28e-7541-11e2-8edf-e0b019be897a.jpg)

**Comment by gasperjug** [comment]
Non-existing nodes might be the issue here. To check which nodes are available the user can dump the platform tree....

We wanted to show the:
Temperature 1 should read the value of the FPGA on the RAF module.
Temperature 2 should read the value of the PLX switch
Temperature 3 should read the value of the EvRX temperature sensor

Can you please check if the nodes are present?

Temperatures:
Command that gives the reading of the temperature:
_libera-bmc --read --sensor | grep FPGA_
Command that gives the temperature from the PCIe switch on ICB:
_libera-bmc --read --sensor | grep PLX_

For the reading of the temperature on EvRX it was chosen R328 sensor close to the clock switch.
The command for the temperature sensor reading:
_libera-bmc --read --sensor | grep CDCE_
